contracts/Holdable.sol

```diff
@@ -0,0 +1,225 @@
+        _totalHeldBalance = _totalHeldBalance.sub(holds[operationId].amount);
+
+        if(block.timestamp >= holds[operationId].expiration && holds[operationId].expiration != 0){
+            holds[operationId].status = HoldStatusCode.ReleasedOnExpiration;
+            emit HoldReleased(holds[operationId].issuer, operationId, HoldStatusCode.ReleasedOnExpiration);
+        }
+        if(block.timestamp < holds[operationId].expiration || holds[operationId].expiration == 0){
+            if(holds[operationId].notary == msg.sender){
+                holds[operationId].status = HoldStatusCode.ReleasedByNotary;
+                emit HoldReleased(holds[operationId].issuer, operationId, HoldStatusCode.ReleasedByNotary);
+            }
+            if(holds[operationId].target == msg.sender){
+                holds[operationId].status = HoldStatusCode.ReleasedByPayee;
+                emit HoldReleased(holds[operationId].issuer, operationId, HoldStatusCode.ReleasedByPayee);
+            }
+        }
+        return true;
+
+    }
+
+
+    function executeHold(string calldata operationId, uint256 value) external returns (bool){
+
+        require(holds[operationId].status == HoldStatusCode.Ordered, "This hold has already been released or executed");
+        require(block.timestamp < holds[operationId].expiration || holds[operationId].expiration == 0, "This hold has already expired");
+        require(holds[operationId].notary == msg.sender, "The hold can only be executed by the notary");
+        require(value <= holds[operationId].amount, "The value should be equal or lower than the held amount");
+
+        heldBalance[holds[operationId].origin] = heldBalance[holds[operationId].origin].sub(holds[operationId].amount);
+        _totalHeldBalance = _totalHeldBalance.sub(holds[operationId].amount);
+
+        _transfer(holds[operationId].origin, holds[operationId].target, value);
+
+        holds[operationId].status = HoldStatusCode.Executed;
+
+        emit HoldExecuted(holds[operationId].issuer, operationId, holds[operationId].notary, holds[operationId].amount, value);
+        return true;
+    }
+
+
+    function renewHold(string calldata operationId, uint256 timeToExpiration) external returns (bool){
+
+        require(holds[operationId].status == HoldStatusCode.Ordered, "This hold has already been released or executed");
+        require(block.timestamp < holds[operationId].expiration || holds[operationId].expiration == 0, "This hold has already expired");
+        require(holds[operationId].origin == msg.sender || holds[operationId].issuer == msg.sender, "The hold can only be renewed by the issuer or the payer");
+
+        uint256 oldExpiration = holds[operationId].expiration;
+        
+        if(timeToExpiration == 0){
+            holds[operationId].expiration = 0;
+        }else{
+            holds[operationId].expiration = block.timestamp.add(timeToExpiration);
+        }
+
+        holds[operationId].expiration = timeToExpiration;
```

Comment:
This line has to be removed, because `holds[operationId].expiration` is set correctly in the if / else statement